### PR TITLE
[V6.1 RC] Update of Fakhari multiphase model

### PR DESCRIPTION
**Pull request by TravisMitchell**
I (minorly) modified the model d2q9_pf_pressureEvolution (the current model in main repo uses a centered difference, this is unable to recover momentum eqn at high density ratio interface).

- Update of naming nu to viscosity for clarity. And update of default value to correspond to omega = 1.
- Update of finite difference stencil used to calculate the gradient of the order parameter. Previous stencil (centered difference) is unable to capture the interface jump, mixed difference required. This increases run time as information from 2 nodes away is required, but means that NS equations are satisfied. If centered difference used, oscillatory behaviour of the velocity profile at an interface and incorrect velocity profile in low density regions are observed. With mixed difference, layered Poiseuille, Rayleigh-Taylor instability, planar bubble rise and elongated planar bubble rise tests have been completed.

### CODE REVIEW

File: models/multiphase/d2q9_pf_pressureEvolution/Dynamics.R [hunk 1]
```diff
@@ -58,8 +58,8 @@ AddSetting(name="sigma", 		   comment='surface tension')
 # 	Inputs: Fluid Properties
 AddSetting(name="omega_l", comment='one over relaxation time (low density fluid)')
 AddSetting(name="omega_h", comment='one over relaxation time (high density fluid)')
-AddSetting(name="nu_l", omega_l='1.0/(3*nu_l)', default=0.16666666, comment='kinematic viscosity')
-AddSetting(name="nu_h", omega_h='1.0/(3*nu_h)', default=0.16666666, comment='kinematic viscosity')
+AddSetting(name="Viscosity_l", omega_l='1.0/(3*Viscosity_l)', default=0.33333333, comment='kinematic viscosity')
+AddSetting(name="Viscosity_h", omega_h='1.0/(3*Viscosity_h)', default=0.33333333, comment='kinematic viscosity')
```
Comment (mdzik) [inline models/multiphase/d2q9_pf_pressureEvolution/Dynamics.R]: Is 0.33333 a reasonable default? why not 1/6?

File: models/multiphase/d2q9_pf_pressureEvolution/Dynamics.R [hunk 2]
```diff
@@ -58,8 +58,8 @@ AddSetting(name="sigma", 		   comment='surface tension')
...
 AddSetting(name="S0", default=1.0, comment='Relaxation Param')
 AddSetting(name="S1", default=1.0, comment='Relaxation Param')
 AddSetting(name="S2", default=1.0, comment='Relaxation Param')
```
Comment (mdzik) [inline models/multiphase/d2q9_pf_pressureEvolution/Dynamics.R]: Do You need/use those?

File: models/multiphase/d2q9_pf_pressureEvolution/Dynamics.c.Rt [hunk 3]
```diff
@@ -4,16 +4,6 @@
...
-
-// Code updates:
-//		19/07/16 - No dynamic grid refinement used, possibly discuss with Lukasz/Michal.
-//		05/08/16 - Typo error found in getMu - fixed
-//				 - MRT relaxation - new variables defined so that matrix multiplication not computed while updating R[0], R[1] etc.
-//      07/10/16 - Verification testing indicated an issue with the model (consistent factor of 2 over prediction)
-//				 - It was found that an error was made in the referenced paper in the given formula for the calculation
-//   			 - of macroscopic velocity. An additional c_s^2 was required on the forcing term.
-  
-
```
Comment (mdzik) [inline models/multiphase/d2q9_pf_pressureEvolution/Dynamics.c.Rt]: Why did you remove that?

File: models/multiphase/d2q9_pf_pressureEvolution/Dynamics.R (hunk 1 above)
Comment (TravisMitchell) [inline models/multiphase/d2q9_pf_pressureEvolution/Dynamics.R]: I had just put as 1/3 quite awhile ago, I think I had been just doing it to set omega to 1. But sorry, I agree 1/6 is more reasonable as corresponds to 1.0 relaxation. I will change and re-commit.

File: models/multiphase/d2q9_pf_pressureEvolution/Dynamics.R (hunk 2 above)
Comment (TravisMitchell) [inline models/multiphase/d2q9_pf_pressureEvolution/Dynamics.R]: Just confirming this is corresponding to the relaxation parameters? I have not played with adjusting any relaxation parameters - had just been my initial set up for the implementation. I will comment them out in the update commit.

File: models/multiphase/d2q9_pf_pressureEvolution/Dynamics.c.Rt (hunk 3 above)
Comment (TravisMitchell) [inline models/multiphase/d2q9_pf_pressureEvolution/Dynamics.c.Rt]: Sorry that one I should have kept, I had been deleting the comments that were above that one as they were unnecessary - but I will re-include the comment regarding the error in the paper.

**Comment by llaniewski** [comment]
@mdzik Please update your review after changes.
